src/test/groovy/pl/tomaszdziurko/jvm_bloggers/SpringContextAwareSpecification.groovy

```diff
@@ -1,19 +1,22 @@
 package pl.tomaszdziurko.jvm_bloggers
 
+import javax.transaction.Transactional
+
 import org.springframework.boot.test.SpringApplicationContextLoader
 import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.context.ContextConfiguration
 
-import spock.lang.Specification;
+import spock.lang.Specification
 
 @ContextConfiguration(classes = [JvmBloggersApplication], loader = SpringApplicationContextLoader)
 @ActiveProfiles("test")
+@Transactional
```

Comment:
Shouldn't we add also following annotation `@TestExecutionListeners(listeners = { TransactionalTestExecutionListener.class })`?